src/errors.ts

```diff
@@ -0,0 +1,8 @@
+export class AccountNotFoundError extends Error {
+  static kind = "AccountNotFound";
```

Comment:
never saw before such thing as:

```
static kind = "AccountNotFound";
public kind = "AccountNotFound";
```

what does that?

also why `this.kind = "AccountNotFound";` in the constructor, that's the same as above?

thanks in advance for the explanation, I am missing something here.

Reply:
Supporting both `AccountNotFoundError.kind` and `this.kind` makes sense to me, however it would be nice if the string did not have to be repeated.  Does `public kind = AccountNotFoundError.kind;` work, for example?